adcm_client/base.py

```diff
@@ -49,6 +49,10 @@ class ObjectNotFound(Exception):
     pass
 
 
+class ActionHasIssues(Exception):
```

Comment:
There is no way for Action to have issues. It should be object has issues.

Reply:
@acmnu 

My fist idea was same. But when i tried to inspect error object to see how to work with it i got

```python
# host that is created from provider without configuration
host = client.host_list()[0]
host.action(name='init').run()
from  coreapi.exceptions import ErrorMessage
try:
    host.action(name='init').run()
except ErrorMessage as error:
    print(error.error)
```

output 
```python
ErrorMessage: <Error: 409 Conflict>
    args: {
        provider: [
            {
                id: "1"
                issue: {
                    config: "False"
                }
                name: "t"
            }
        ]
    }
    code: "TASK_ERROR"
    desc: "action has issues"
    level: "error"
```

In description is said  **action has issues**